### ISSUE TITLE
Unable to import xmlrpc with cx_freeze executable

### ISSUE THREAD

**Issue by Lois31** [marcelotduarte/cx_Freeze#768]
When I freeze my application, I get the following error when launching the executable:

> Traceback (most recent call last):
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/site-packages/cx_Freeze/initscripts/__startup__.py", line 40, in run
    module.run()
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/site-packages/cx_Freeze/initscripts/Console.py", line 37, in run
    exec(code, {'__name__': '__main__'})
  File "test-cli.py", line 1, in <module>
    from xmlrpc.server import SimpleXMLRPCServer, SimpleXMLRPCRequestHandler
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/xmlrpc/server.py", line 117, in <module>
    import pydoc
  File "<frozen zipimport>", line 259, in load_module
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/pydoc.py", line 366, in <module>
    class Doc:
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/pydoc.py", line 396, in Doc
    def getdocloc(self, object, basedir=sysconfig.get_path('stdlib')):
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/sysconfig.py", line 513, in get_path
    return get_paths(scheme, vars, expand)[name]
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/sysconfig.py", line 503, in get_paths
    return _expand_vars(scheme, vars)
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/sysconfig.py", line 172, in _expand_vars
    _extend_dict(vars, get_config_vars())
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/sysconfig.py", line 551, in get_config_vars
    _init_posix(_CONFIG_VARS)
  File "/home/ioland/REF/COTS/PYTHON/V3.8.3/BINARIES/Linux/lib/python3.8/sysconfig.py", line 422, in _init_posix
    _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
ModuleNotFoundError: No module named '_sysconfigdata__linux_x86_64-linux-gnu'

I am using the following cx_freeze configuration: [cx_freeze_conf.zip](https://github.com/marcelotduarte/cx_Freeze/files/5402664/cx_freeze_conf.zip)

Platform configuration:

- Linux RHEL 7
- Python V3.8.3
- cx_Freeze 6.1

I fixed the problem by manually importing the missing file by adding the following lines in my _setup.py_ file:

```
import sysconfig
include_files = []
if sys.platform == 'linux':
    include_files.append(os.path.join(os.path.dirname(sysconfig.__file__), '_sysconfigdata__linux_x86_64-linux-gnu.py'))
```

**Comment by marcelotduarte** [comment]
cx_Freeze 6.3 has just been released.
PR #727 resolves this issue.
If you had issues please re-open.

**Comment by Lois31** [comment]
This problem is fixed with cx_Freeze 6.3.
